src/huggingface_hub/hf_api.py

```diff
@@ -155,6 +155,7 @@ def list_models(
         direction: Optional[Literal[-1]] = None,
         limit: Optional[int] = None,
         full: Optional[bool] = None,
+        config: Optional[bool] = None,
```

Comment:
WDYT of renaming `config` to `fetch_config`? Put like that I'd expect it to be a configuration parameter, rather than a request to fetch the configurations

Reply:
Great idea! Done